compiler/src/dmd/id.d

```diff
@@ -223,15 +223,15 @@ immutable Msgtable[] msgtable =
     { "__LOCAL_SIZE" },
 
     // For operator overloads
-    { "eq",      "opEquals" },
-    { "cmp",     "opCmp" },
-    { "assign",  "opAssign" },
-    { "index",   "opIndex" },
-    { "indexass", "opIndexAssign" },
-    { "slice",   "opSlice" },
-    { "sliceass", "opSliceAssign" },
-    { "call",    "opCall" },
-    { "_cast",    "opCast" },
+    { "opEquals" },
+    { "opCmp" },
+    { "opAssign" },
+    { "opIndex" },
+    { "opIndexAssign" },
+    { "opSlice" },
+    { "opSliceAssign" },
+    { "opCall" },
+    { "opCast" },
```

Comment:
Need to update `id.h` too. Also probably `frontend.h`

Reply:
I updated frontend.h, but id.h contains just:

```D
#pragma once

struct Id
{
    static void initialize();
};
```